libs/designsystem/dropdown/src/dropdown.component.ts

```diff
@@ -525,6 +524,104 @@ export class DropdownComponent implements AfterViewInit, OnDestroy, ControlValue
     }
   }
 
+  private preventDefaultAndStopImmediatePropagation(event: KeyboardEvent) {
+    event.stopImmediatePropagation();
+    event.preventDefault();
+  }
+
+  private searchBuffer = '';
+  private searchBufferTimeout: any;
+  private SEARCH_BUFFER_DELAY = 500; // ms
+
+  private addToSearchBuffer(char: string) {
+    clearTimeout(this.searchBufferTimeout);
+    this.searchBuffer += char;
+    this.searchBufferTimeout = setTimeout(() => {
+      this.resetSearchBuffer();
+    }, this.SEARCH_BUFFER_DELAY);
+  }
+  private resetSearchBuffer() {
+    this.searchBuffer = '';
+  }
+
+  @HostListener('keydown', ['$event'])
+  _onKeydown(event: KeyboardEvent): boolean {
+    const key = event.key;
+
+    if (this.items.length === 0) {
+      console.warn('[Kirby] No items found within dropdown');
+      return false;
```

Comment:
"false" doesn't mean anything, so lets just do `return;`
```suggestion
      return;
```